Source/WebCore/dom/TrustedTypePolicy.h

```diff
@@ -49,6 +50,7 @@ class TrustedTypePolicy : public ScriptWrappable, public RefCounted<TrustedTypeP
     ExceptionOr<Ref<TrustedHTML>> createHTML(const String& input, FixedVector<JSC::Strong<JSC::Unknown>>&&);
     ExceptionOr<Ref<TrustedScript>> createScript(const String& input, FixedVector<JSC::Strong<JSC::Unknown>>&&);
     ExceptionOr<Ref<TrustedScriptURL>> createScriptURL(const String& input, FixedVector<JSC::Strong<JSC::Unknown>>&&);
+    ExceptionOr<String> getPolicyValue(const TrustedType trustedTypeName, const String& input, FixedVector<JSC::Strong<JSC::Unknown>>&&, bool throwIfMissing = true);
```

Comment:
Should just be `TrustedType` not `const TrustedType`.

We try to avoid mystery `bool` arguments like this `throwIfMissing` one. Can we find a way to not have that, perhaps by having more than one function? I think one function can return null string if missing and the one that throws if missing can just call the other and post process it.

Reply:
The problem with returning nullString and post processing is that the callbacks themselves can return null and they shouldn't throw in this situation.

I'll make it a boolean enum instead so it's more explicit.

Reply:
I think you can split this into two separate functions. Whether internally you want to use a `bool` argument to a common function or can find a way to have one call the other is something we can tinker with. I am almost certain you can have one call the other but I can propose a specific way once I see your revised patch. For example if you can distinguish the specific exception you can convert the exception to a null string.

Reply:
Yes clearly wrong if it needs to check the string.

Reply:
I've changed it to an enum so it's at least not a magic bool.